kitti_rec:
  ros__parameters:
    sequence: 2
    data_dir: '/media/psf/SSD/DRONES_LAB/kitti_dataset/dataset'
    odom_dir: '/media/psf/SSD/DRONES_LAB/kitti_dataset/dataset_2'
    bag_dir : '/home/parallels/Desktop/bag'
    odom : True